### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by LukeWood** [keras-team/keras-cv#316] [Show and tell]
This is KerasCV's v0.1.0 release!

This is the first official release of KerasCV.  Please note there is no backwards compatibility contract until KerasCV reaches v1.0.0.  

In this release KerasCV introduces:
- 19 new image data augmentation layers
- a Keras native RandAugment implementation
- fully in graph COCO metrics for bounding box detection!

Keep an eye out on keras.io in the upcoming weeks for our new API documentation and guides.

Thank you to all of the wonderful contributors that have made this release possible!

<hr /><em>This discussion was created from the release <a href='https://github.com/keras-team/keras-cv/releases/tag/'>v0.1.0</a>.</em>

**Comment by LukeWood** [comment]
Personal thank you from myself to all of our community contributors: @chjort @sebastian-sz @innat @quantumalaviya @winstoncooke @bhack great job making contributions to this repo!

**Reply by 39kikurage** [reply]
I am excited about this release! Thank you.
